parsely/build.gradle

```diff
@@ -22,6 +33,7 @@ android {
         exclude 'META-INF/LICENSE'
         exclude 'META-INF/NOTICE'
     }
+    testBuildType "debug"
```

Comment:
Question (❓): I noticed this, plus the fact that you are using `./gradlew :parsely:connectedCheck` on CI for the functional tests, why is that? I mean, why not doing exactly what've done for the unit test, using the `Debug` variant for the task name, like `connectedDebugAndroidTest` as using the `connectedCheck` is a super task that incorporate more checks than just running the Android instrumentation tests, which is what we want here, right?

FYI: This way, removing this constraint, it will also enable us to use the `release` build type in the future, should we need it for Android tests, right? 🤔

PS: This is just an FYI question that I want to understand more, not to question it's validity, I am sure there is a good reason you used `connectedCheck` here. 🙏

Reply:
Good point, thanks! Actually, there's no really a reason why I used `connectedCheck`: fixed this in https://github.com/Parsely/parsely-android/pull/85/commits/d2a25dbc864396d67dc517943a5a079cffb86304 🙂 👍

Reply:
Awesome, thanks for this change!

Btw, now that you did d2a25dbc864396d67dc517943a5a079cffb86304 , do we actually need this `testBuildType "debug"` configuration? 🤔

Reply:
See e6727a0bd213dd72cad11c45988142894c237d42 and thus resolving this, thanks!